base/complex.jl

```diff
@@ -464,7 +464,7 @@ function inv(w::ComplexF64)
     return ComplexF64(p*s,q*s) # undo scaling
 end
 
-function ssqs(x::T, y::T) where T<:AbstractFloat
+function ssqs(x::T, y::T) where T
```

Comment:
Maybe Real or Number?

Reply:
This is already an internal function, so I don't think we need to constraint the type just to make it more flexible.